test/putTransferSpec.js

```diff
@@ -688,7 +701,7 @@ describe('PUT /transfers/:id', function () {
         }))
         .end()
     })
-
+/*
```

Comment:
The transfer can only be executed when posting to the `/transfers/:id/fulfillment` endpoint. An authorized transfer with no `execution_condition` will no longer execute automatically. Let's discuss, and decide if we want this behavior.